test/e2e/volume_provisioning.go

```diff
@@ -122,25 +122,9 @@ var _ = framework.KubeDescribe("Dynamic provisioning", func() {
 			testDynamicProvisioning(c, claim)
 		})
 	})
-
-	framework.KubeDescribe("DynamicProvisioner Alpha", func() {
```

Comment:
I am not sure I can add/remove a default storage class in an e2e test - there already may be one or the admission plugin may be disabled in that particular Kubernetes deployment.